templates/education/pdce-workshops.php

```diff
@@ -11,16 +11,21 @@
 
 $the_title          = get_the_title();
 
-// Some custom fields
-$coursecodeundergrad			= get_post_meta( absint( $post_id ), 'coursecodeundergrad', true );
-$coursecodegrad					= get_post_meta( absint( $post_id ), 'coursecodegrad', true );
-$vanityurl 						= get_post_meta( absint( $post_id ), 'vanityurl', true );
-$credit 						= get_post_meta( absint( $post_id ), 'credit', true );
-$promophoto 					= get_post_meta( absint( $post_id ), 'promophoto', true );
-$subtitle						= get_post_meta( absint( $post_id ), 'subtitle', true );
-$dates							= get_post_meta( absint( $post_id ), 'dates', true );
-$location						= get_post_meta( absint( $post_id ), 'location', true );
+//Get the post permalink
+
+$permalink          = get_permalink( $post );
```

Comment:
Is this meant to be `$post_id` rather than `$post`? The `get_permalink()` function can actually take a `WP_Post` Object, but it seems a little odd here in this context.